photocollage/gtkgui.py

```diff
@@ -20,6 +20,10 @@
 import cairo
 import copy
 import gettext
+import gi
+
+gi.require_version('Gtk', '3.0')
```

Comment:
OK, I see... In this case, the first version of you PR was good (as it kept the imports sorted).

Could you revert to this:

```py
import gettext
import gi
gi.require_version('Gtk', '3.0')
from gi.repository import Gtk, Gdk, GObject		
from io import BytesIO
```

Then it will be good to go :)